.buildkite/pipeline.yml

```diff
@@ -23,42 +26,58 @@ steps:
             # Allow WP-CLI to be run as root, otherwise it throws an exception.
             # Reference: https://git.io/J9q2S
             - "WP_CLI_ALLOW_ROOT=true"
+      - *git-shallow-clone
     command: |
         source /root/.bashrc
 
+        echo "--- :node: Setup Node environment"
         nvm install && nvm use
+
+        echo "--- :npm: Install Node dependencies"
         npm ci --unsafe-perm --prefer-offline --no-audit --no-progress
 
+        echo "--- :package: Run bundle prep work"
         npm run prebundle:js
+
+        echo "--- :android: Build Android bundle"
         npm run bundle:android
 
+        echo "--- :arrow_up: Upload Android bundle artifact"
         buildkite-agent artifact upload bundle/android/App.js
 
+        echo "--- :ios: Build iOS bundle"
+        npm run bundle:ios
+
+        echo "--- :arrow_up: Upload iOS bundle artifact"
+        buildkite-agent artifact upload bundle/ios/App.js
+
   - label: "Build Android RN Aztec & Publish to S3"
     key: "publish-react-native-aztec-android"
     plugins:
       - *publish-android-artifacts-docker-container
+      - *git-shallow-clone
     command: |
         .buildkite/publish-react-native-aztec-android-artifacts.sh
 
   - label: "Build Android RN Bridge & Publish to S3"
     depends_on:
-      - "bundle-android"
+      - "js-bundles"
       - "publish-react-native-aztec-android"
     plugins:
       - *publish-android-artifacts-docker-container
+      - *git-shallow-clone
     command: |
         .buildkite/publish-react-native-bridge-android-artifacts.sh
 
   - label: Build iOS RN XCFramework
+    depends_on: js-bundles
     command: .buildkite/publish-react-native-ios-artifacts.sh
     artifact_paths:
       - ios-xcframework/build/xcframeworks/*.zip
     plugins:
       - automattic/a8c-ci-toolkit#2.15.1
-      - peakon/git-shallow-clone#v0.0.1:
-          depth: 1
+      - *git-shallow-clone
     agents:
       queue: mac
     env:
-      IMAGE_ID: xcode-14.2
+      IMAGE_ID: xcode-14.3
```

Comment:
Considering we are updating Xcode, may as well update it to the latest one `14.3.1-rc.1` which is available on CI. 😸

Reply:
During development, I experienced an issue which was resolved, I think, by putting the builder and consumer projects on the same Xcode version 😳 

For the moment, I'd like to keep the two in sync.